sdk/platform/http_client/curl/src/curl_client.cpp

```diff
@@ -99,6 +99,10 @@ static void ParseHeader(std::string const& header, std::unique_ptr<Response>& re
 
   auto headerName = std::string(start, end);
   start = end + 1; // start value
+  while (start < header.end() - 2 && (*start == ' ' || *start == '\t'))
```

Comment:
What does the `-2` here mean? Maybe a comment would be helpful.

Also, @vhvb1989 - we should certainly have tests for these :)

Reply:
@ahsonkhan I removed the ` -2` so it's more intuitive.